### ISSUE TITLE
Correct Link to Git Gui Wiki from Contributing.md

### ISSUE THREAD

**Issue by frankstepanski** [hackforla/website#2673]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
The link to the Git Gui Wiki from How to Contribute to Hack for LA needs correction since the current link is broken.

### Action Items
- [ ] Change the **Git Gui Wiki** link (under [2.7.c Working on an issue (3): Prepare your changes to push to your repository](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27c-working-on-an-issue3-prepare-your-changes-to-push-to-your-repository))
From
`https://github.com/hackforla/website/wiki/Using-Git-GUI-(Graphical-user-Interface)-in-Visual-Studios-Code`
To
`https://github.com/hackforla/website/wiki/How-to-Use-Git-GUI-(Graphical-user-Interface)-in-Visual-Studios-Code`

### Resources/Instructions
- The correct link is:
[https://github.com/hackforla/website/wiki/How-to-Use-Git-GUI-(Graphical-user-Interface)-in-Visual-Studios-Code](https://github.com/hackforla/website/wiki/How-to-Use-Git-GUI-(Graphical-user-Interface)-in-Visual-Studios-Code)
- File where changes need to be made: [Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md) 
- File location in our codebase: `website/CONTRIBUTING.md`

**Comment by MukulKolpe** [comment]
Hey @frankstepanski can I work on this issue?

**Comment by frankstepanski** [comment]
@MukulKolpe Hi, this issue first needs to be reviewed by Bonnie and moved to the Prioritized Backlog Column before it can be assigned. I probably will assign it to myself afterward.

**Comment by ExperimentsInHonesty** [comment]
@MukulKolpe 
We require all Code Contributors to
1. Join our organization by going through [Hack for LA onboarding](https://www.hackforla.org/getting-started).  It's free to join!
2. Read the [onboarding section of our WIKI](https://github.com/hackforla/website/wiki/Joining-the-HackforLA.org-Website-Team).
3. Read our [CONTRIBUTING](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md) document and follow the steps.

**Comment by frankstepanski** [comment]
Availability for this week:
My estimated ETA for completing this issue: Today

UPDATE
Progress: Have submitted a pull request. Awaiting approval
Blockers: None
Availability: Weeknights (EST) and Sunday
ETA: Completed
Pictures: https://github.com/hackforla/website/pull/2741

**Comment by SAUMILDHANKAR** [comment]
@frankstepanski: Hi Frank, Thanks for working on this issue and providing ETA and availability. Please note the following while working on your next issue:
  - The issue should remain open until the PR has been merged. Infact, once the PR is merged the issue is closed automatically. So, I am going to reopen this issue now.

**Comment by SAUMILDHANKAR** [comment]
@frankstepanski change not implemented in section 2.7.c in the contributing.MD, so reopening this issue.

**Comment by ExperimentsInHonesty** [comment]
Hi @frankstepanski when Providence was doing the QA (what happens after an issue gets merged) we determined that this issue was not completed.  Please fix and submit a new pr.

**Comment by blulady** [comment]
@frankstepanski are you still working on this?

**Comment by ExperimentsInHonesty** [comment]
Dev leads, please review the issue and determine what remains to be done, revise the issue, leave a note about what was changed and then add ready for milestone label.

**Comment by jdingeman** [comment]
Hello team. I just looked through this issue and located the "broken" link. It looks like the link in [2.7 Working on an issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) does lead to `https://github.com/hackforla/website/wiki/How-to-Use-Git-GUI-(Graphical-user-Interface)-in-Visual-Studios-Code` and is not broken, however the link under [2.7.c Working on an issue (3): Prepare your changes to push to your repository](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27c-working-on-an-issue3-prepare-your-changes-to-push-to-your-repository) is broken.
I have removed the first one and left the second one as the link needing to be corrected.
